src/plugins/template/tests/functional/op_reference/adaptive_avg_pool.cpp

```diff
@@ -28,8 +28,8 @@ struct AdaptiveAvgPoolParams {
           m_output_shape(output_shape),
           m_input_type(input_type),
           m_output_type(ouput_type),
-          m_input_data(CreateTensor(input_type, input_values)),
```

Comment:
Why it is required to specify shape? Is flat tensors comparison not working correct?

Reply:
`ov::test::utils::compare()` [compares shapes](https://github.com/openvinotoolkit/openvino/blob/12481984f74abd2ff00ec172c397075d518705b3/src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp#L419).